src/Analysis/GWASUIApp/TourButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from "react";
+import { useTour } from '@reactour/tour'
+import { case_control_steps, quantitative_steps } from "./gwassteps";
+
+const TourButton = ({stepInfo}) => {
+    const { setIsOpen, setSteps, isOpen } = useTour()
+
+    let current_steps = stepInfo.workflowName === "case control" ? case_control_steps : quantitative_steps
+
+    useEffect(() => {
+        setSteps(current_steps[stepInfo.step])
+    }, [stepInfo])
+
+
+    return (<button onClick={() => {
+        if (!isOpen) {
+            setIsOpen(true)
+        }
+    }}>Open Tour</button>)
```

Comment:
instead of "Open Tour" text here, could you add a field to stepInfo (ex. `stepInfo.tourButtonLabel`) to populate this for reusability. see other comment